runtime/packages/models/assembly/extrinsic/signed-transaction.ts

```diff
@@ -74,32 +78,36 @@ export class Extrinsic {
      */
     static fromU8Array(input: u8[]): DecodedData<Extrinsic> {
         assert(input.length >= 144, "Extrinsic: Invalid bytes provided. EOF");
+        // get only signed transaction bytes
+        let signedTxu8a = input.slice(0, ExtrinsicType.SignedTransaction);
+        
+        input = input.slice(ExtrinsicType.SignedTransaction);
```

Comment:
Normally in the other `fromU8Arrays` we do the following:
1. Read the properties one after another (f.e from, to, amount, nonce, signature, exchausResourcesWhenNotFirst)
2. We return what's left afterwards.

Wouldn't it be possible to do it the same way here as-well ?

Reply:
fixed it for `SignedTransaction`, but for `Inherent` `arg` property comes as sliced (two last bytes are sliced). That's why, we need to fill the array's two last bytes with zeros.